### PR TITLE
Update the serial port part of UP2's doc

### PR DESCRIPTION
**Pull request by coneypo**
Update the doc of "Getting started guide for UP2 board" - "Hardware setup";
(https://github.com/projectacrn/acrn-hypervisor/blob/master/doc/getting-started/up2.rst)

1.  Update the link of "UP2 Datasheet";
2.  Add the images of UP2 board, serial port cable and the conncetion;
3.  Update the introduction of connections between UP2 board and serial port cable.

Signed-off-by: Xie, zhengtian zhengtian.xie@intel.com

### CODE REVIEW

**Comment by coneypo** [comment]
The updated doc: https://github.com/coneypo/acrn-hypervisor/blob/doc_of_UP2/doc/getting-started/up2.rst

File: doc/getting-started/up2.rst [hunk 1]
```diff
@@ -19,16 +19,30 @@ SoCs. Both have been confirmed to work with ACRN.
...
-specifications <http://www.up-board.org/upsquared/specifications-up2/>`_
-for more information.  We'll access the serial port through the I/O pins
-in the 40-pin HAT connector using a `USB TTL serial cable
-<http://www.ftdichip.com/Products/USBTTLSerial.htm>`_. Connect pin 6
-(``GND``), pin 8 (``TX``) and pin 10 (``RX``) of the HAT connector to
-respectively the ``GND``, ``RX`` and ``TX`` pins of your USB serial
-cable. Plug the USB TTL serial cable into your PC and use a console
-emulation tool such as ``minicom`` or ``putty`` to communicate with the
-UP2 board for debugging.
+The UP2 board has two serial ports and the following figure shows the bottom-side of UP2 board,
```
Comment (dbkinder) [inline doc/getting-started/up2.rst]:
```suggestion
The UP2 board has two serial ports. The following figure shows the
UP2 board's 40-pin HAT connector we'll be using, 
```

File: doc/getting-started/up2.rst [hunk 2]
```diff
@@ -19,16 +19,30 @@ SoCs. Both have been confirmed to work with ACRN.
...
-for more information.  We'll access the serial port through the I/O pins
-in the 40-pin HAT connector using a `USB TTL serial cable
-<http://www.ftdichip.com/Products/USBTTLSerial.htm>`_. Connect pin 6
-(``GND``), pin 8 (``TX``) and pin 10 (``RX``) of the HAT connector to
-respectively the ``GND``, ``RX`` and ``TX`` pins of your USB serial
-cable. Plug the USB TTL serial cable into your PC and use a console
-emulation tool such as ``minicom`` or ``putty`` to communicate with the
-UP2 board for debugging.
+The UP2 board has two serial ports and the following figure shows the bottom-side of UP2 board, 
+which is from `UP2 specifications
```
Comment (dbkinder) [inline doc/getting-started/up2.rst]:
```suggestion
as documented in the `UP2 Datasheet 
```

File: doc/getting-started/up2.rst [hunk 3]
```diff
@@ -19,16 +19,30 @@ SoCs. Both have been confirmed to work with ACRN.
...
+which is from `UP2 specifications 
+<https://up-board.org/wp-content/uploads/datasheets/UP-Square-DatasheetV0.5.pdf>`_.
+
+.. image:: images/the-bottom-side-of-UP2-board.png
+   :align: center
+   
+We'll access the serial port through the I/O pins in the 
+40-pin HAT connector using a `USB TTL serial cable
+<http://www.ftdichip.com/Products/USBTTLSerial.htm>`_, 
+and introduce how to enable serial port with
```
Comment (dbkinder) [inline doc/getting-started/up2.rst]:
```suggestion
and show how to connect a 
```

**Comment by dbkinder** [comment]
There seem to be a lot of unrelated commits in this PR?  Can you clean this up when you submit the recommended changes.

**Pull request merged**